src/amo/client.js

```diff
@@ -7,11 +10,18 @@ import App from './components/App';
 import sagas from './sagas';
 import createStore from './store';
 
-createClient(createStore, { sagas }).then(({ renderApp }) => {
+createClient(createStore, { sagas }).then(({ history, renderApp, store }) => {
   renderApp(App);
 
   if (module.hot) {
-    module.hot.accept('./components/App', () => {
+    module.hot.accept(['./sagas', './store'], () => {
```

Comment:
It would be nice to see a comment here explaining that sagas aren't hot reloadable but this is needed to facilitate reducer/store reloading.